packages/utilities/etc/utilities.api.md

```diff
@@ -18,7 +18,7 @@ export function addDirectionalKeyCode(which: number): void;
 export function addElementAtIndex<T>(array: T[], index: number, itemToAdd: T): T[];
 
 // @public
-export const allowScrollOnElement: (element: HTMLElement | null, events: EventGroup) => void;
+export const allowScrollOnElement: (element: HTMLElement | null, events: EventGroup, allowIosOverscroll?: boolean) => void;

```

Comment:
Adding a boolean arg is usually not recommended - can we circumvent this without a boolean arg?

Reply:
See: https://github.com/OfficeDev/office-ui-fabric-react/wiki/Pull-request-reviewing-guidance
> Avoid using boolean params in method definitions. Boolean params usually mean a conditional if statement will be used which means the caller probably knows too much about the implementation of the method it's calling. It also makes it hard to change the method later. Make 2 methods that do different things instead.

Reply:
Originaly i tought of this, but this would create duplicate code. It would be just 2 slighty different versions of [_makeElementScrollAllower](https://github.com/OfficeDev/office-ui-fabric-react/blob/2894062b1d0515055c29061dfc6f586f918ca41c/packages/utilities/src/scroll.ts#L21). But now i see if you remove the code that is needed for _preventOverscrolling handler, not much is left and duplicate code will be minimal.